app/models/session.rb

```diff
@@ -19,8 +19,9 @@ class Session < Common::RedisStore
   attribute :ssoe_transactionid
   attribute :profile
   attribute :charon_response
+  attribute :launch
+  attribute :uuid
```

Comment:
It might be worth a comment here, indicating that `uuid` is optional since a session may be for a system (vs a user). I worry that someone may revert this change without as thorough of a review... breaking us.